### PR TITLE
Fix #143 : sort the headers alphabetically for `fortio server` /debug endpoint

### LINKED ISSUE #143: fortio /debug : sort the headers
would make it easier to find

### PR DESCRIPTION
**Pull request by SeriyBg**
fixes #143

### CODE REVIEW

File: fhttp/http.go [hunk 1]
```diff
@@ -35,6 +35,8 @@ import (
 	// get /debug/pprof endpoints on default server - make sure report only doesn't have it
 	_ "net/http/pprof"
 
+	"sort"
```
Comment (ldemailly) [inline fhttp/http.go]: please put that import with the others (should be sorted automatically bu go fmt)

**Review by ldemailly — COMMENTED** [review]
Thanks ! small nit:

**Comment by ldemailly** [comment]
also put “fixes #143” in the PR description

**Comment by danehans** [comment]
I performed the following test:
```
$ docker run --name client --rm istio/fortio:webtest curl -loglevel verbose \
-H Zab:Baz -H Foo:Bar -H Baz:Barf -curl http://172.17.0.2:8080/debug

<SNIP>
headers:

Host: 172.17.0.2:8080
Baz: Barf
Foo: Bar
User-Agent: istio/fortio-0.7.2-pre
Zab: Baz

<SNIP>
```

@ldemailly are you looking for only the extra headers to be sorted alphabetically or all non-Host headers?

**Comment by ldemailly** [comment]
Host is special so I don't mind we put it first

what do you mean "extra headers" vs "all non Host headers" ? what is the difference?

**Comment by danehans** [comment]
I should have stated "additional headers", i.e. `-H` flag instead of extra headers.

File: fhttp/http.go (hunk 1 above)
Comment (SeriyBg) [reply fhttp/http.go]: Done

**Comment by SeriyBg** [comment]
@ldemailly thanks for your comments! I have fixed it

**Review by ldemailly — DISMISSED** [review]
thank you for your contribution

ideally there would be a unit test for this - do you want to add one ? (see http_test.go)
just add 2 headers A: x, B:y and make sure you get A before B ? maybe run the the check 3 times to ensure you don't get lucky, or add more headers to make it less likely to accidentally pass
(ideally comment out your new code, see the test fail, put back the fix, see the test pass)

**Comment by SeriyBg** [comment]
@ldemailly sure will add the test. 
Just a quick question: is there any mock framework used to build the tests? I actually need to have a `mockResponseWriter` so I can test the `DebugHandler`. Or should I write the test on the upper level? If so, then maybe the test must be located in `httprunner_test`?

**Comment by ldemailly** [comment]
You use a dynamichttp see the tests in http_test that start a server and connect to it. Early tests using the server were in httprunner test but new test should be in http test

So not a mock, a real request and response (kind of e2e test in unit for)

**Comment by ldemailly** [comment]
an example of test/where to add:
https://github.com/istio/fortio/blob/master/fhttp/http_test.go#L564

**Comment by SeriyBg** [comment]
@ldemailly thanx for your help. I have added the test for this issue, you can take a look when you have time.

**Review by ldemailly — APPROVED** [review]
awesome ! thanks !

**Pull request merged**